src/canvas-effects/canvas-effect.js

```diff
@@ -1767,11 +1551,34 @@ export default class CanvasEffect extends PIXI.Container {
 	}
 
 	/**
-	 * If this effect is attached to an object, check whether the object has been destroyed, if so, end the effect
+	 * Add Ticker handler to check for updates to attached objects
 	 *
 	 * @private
 	 */
-	_contextLostCallback() {
+	_registerTickers() {
+		//stretchTo && attached to stretchTo
+		if (this.data.stretchTo && this.data.stretchTo?.attachTo) {
+			this._addToTicker(this._transformStretchToAttachedSprite);
+		}
+		// attachTo, not attached to stretchTo
+		if (this.data.attachTo?.active && !this.data.stretchTo?.attachTo) {
+			this._addToTicker(this._transformAttachedNoStretchSprite);
+		}
+
+		// rotateTowards
+		if (this.rotateTowards && this.data.rotateTowards?.attachTo) {
+			this._addToTicker(this._transformRotateTowardsAttachedSprite);
+		}
+
+		// scaleTo
+		if (this.data.scaleToObject && this.data?.attachTo?.active && this.data?.attachTo?.bindScale) {
+			this._addToTicker(() => {
+				this._applyScaleToObject(heightWidthRatio, widthHeightRatio, baseScaleX, baseScaleY);
```

Comment:
References to these variables were lost due to the refactor, these were used to keep track of the original dimensions. This currently throws and error and breaks the canvas if this is hit.

Reply:
I will take a look at this. That's unfortunate that I missed it -.-
I'm on vacation atm and can maybe take care of it this week, but maybe I'll only get to it once I'm back in about 9 days or  so...

Reply:
No problemo, I don't think I'll have time to integrate this within the week anyway, given that I'm moving in two days! Thanks for the hard work nonetheless.

Reply:
This is fixed now, with a basic example for attachTo with `bindScale` added.